src/components/celestialmap/skymap_config.js

```diff
@@ -0,0 +1,304 @@
+let base_config = {
+    width: 0,     // Default width, 0 = full parent width; height is determined by projection
+    projection: "stereographic",  // Map projection used: airy, aitoff, armadillo, august, azimuthalEqualArea, azimuthalEquidistant, baker, berghaus, boggs, bonne, bromley, collignon, craig, craster, cylindricalEqualArea, cylindricalStereographic, eckert1, eckert2, eckert3, eckert4, eckert5, eckert6, eisenlohr, equirectangular, fahey, foucaut, ginzburg4, ginzburg5, ginzburg6, ginzburg8, ginzburg9, gringorten, hammer, hatano, healpix, hill, homolosine, kavrayskiy7, lagrange, larrivee, laskowski, loximuthal, mercator, miller, mollweide, mtFlatPolarParabolic, mtFlatPolarQuartic, mtFlatPolarSinusoidal, naturalEarth, nellHammer, orthographic, patterson, polyconic, rectangularPolyconic, robinson, sinusoidal, stereographic, times, twoPointEquidistant, vanDerGrinten, vanDerGrinten2, vanDerGrinten3, vanDerGrinten4, wagner4, wagner6, wagner7, wiechel, winkel3
+    projectionRatio: null, // Optional override for default projection ratio
+    transform: "equatorial", // Coordinate transformation: equatorial (default), ecliptic, galactic, supergalactic
+
+    // Initial center coordinates in equatorial transformation [hours, degrees, degrees],
+    // otherwise [degrees, degrees, degrees], 3rd parameter is orientation, null = default center
+    // NOTE: don't set center here; load in component where site lat/lng is known. 
+    //center: [helpers.hour2degree(helpers.siderealTime(-119)), 34,],       
+    follow: "center",   // on which coordinates to center the map, default: zenith, if location enabled, otherwise center
+
+    orientationfixed: true,  // Keep orientation angle the same as center[2]
+    //geopos: [34,-119],    // optional initial geographic position [lat,lon] in degrees, overrides center
+
+    background: { fill: "#080f17", stroke: " #17202a", opacity: 1 }, // Background style
+    adaptable: false,    // Sizes are increased with higher zoom-levels
+    interactive: false, // Enable zooming and rotation with mousewheel and dragging
+    form: false,        // Display settings form
+    // Set visiblity for each group of fields of the form
```

Comment:
So currently no way to toggle constellations/dsos/etc, but plans to do so?

Reply:
Yes. By the way, this big config file is mostly written around the needs of the d3-celestial library. The builtin form they use is functional but not great from a UI perspective. We'll probably want to write our own.